src/components/spatial/Spatial.js

```diff
@@ -136,14 +148,21 @@ class Spatial extends AbstractSpatialOrScatterplot {
     // so we need to flip it for rendering tooltips.
     const flipYTooltip = true;
 
-    return new SelectablePolygonLayer({
+    return new PolygonLayer({
       id: CELLS_LAYER_ID,
       backgroundColor: [0, 0, 0],
       isSelected: getCellIsSelected,
       getPolygon: getCellPolygon,
       updateTriggers: {
-        getFillColor: [opacity],
         getLineWidth: [stroked],
+        isSelected: cellSelection,
+        getExpressionValue,
```

Comment:
This feels like a cludge - the data should just live in the `data` prop but our architecture is a bit convoluted right now for updating that.  I don't think the performance is different but this is a bit of anti-pattern

Reply:
Though, I suppose this is actually true for all of our accessors. I think this might need to be an issue dealt with separately.  I think the plumbing here needs to be swapped out.

Reply:
The accessor pattern feels fine to me, especially for something like selections which are more dynamic.

Here we are not using `isSelected` or `overlayBaseProps` though.